specs/engage-digital_openapi3.yaml

```diff
@@ -4312,31 +4310,31 @@ paths:
       parameters:
         - description: To filter users on given source ids (based on reply permission).
           in: query
-          name: source_ids
+          name: 'source_ids[]'

```

Comment:
Question for all of them : quotes are needed because you added some brackets ? Is it a yaml rule ? or for the postman parser ?